Dockerfile

```diff
@@ -26,10 +26,10 @@ RUN apt-get update -qq \
 
 # install poetry
 # keep this in sync with the version in pyproject.toml and Dockerfile
-ENV POETRY_VERSION 1.8.2
+ENV POETRY_VERSION=1.8.2
```

Comment:
Is it possible to use poetry version as an env var from [here](https://github.com/RasaHQ/rasa-sdk/blob/main/scripts/poetry-version.sh) instead of hardcoding it? Makes it easier when upgrading poetry.

Reply:
Done.